spef/testing/report.py

```diff
@@ -6,8 +6,17 @@
 from jinja2 import Environment, FileSystemLoader
 
 from spef.testing.tst import calculate_score
-from spef.utils.loading import *
-from spef.utils.logger import *
+from spef.utils.loading import load_report_from_file, load_testsuite_tags
+from spef.utils.match import get_tests_names
+from spef.utils.logger import (
+    log,
+    REPORT_SUFFIX,
+    DATA_DIR,
+    REPORT_TEMPLATE,
+    TESTS_DIR,
+    REPORT_DIR,
+    TOTAL_REPORT_FILE,
+)
```

Comment:
This is perhaps more compact?
```suggestion
import spef.utils.logger as logger
```
Anyways, the same situation with `logger` happens in multiple places and the current code uses both approaches so we should probably unify this. I'd vote for the more compact version.

Reply:
I usually use `from X import x, y, z, ...` when I need to import just a few names from the specific file and `import X as _` when there's too many names to specify manually. This is quite common approach that reduces code verbosity for loosely coupled modules (option 1) while giving you the flexibility to use any name you need from the module without changing the import statement for tightly coupled modules (option 2). If you'd prefer to unify the importing style, then we should probably unify it across all imports, as it would be hard to track which file should be imported in which manner.

Reply:
I think that your approach is fine, I just noticed that specifically the `logger` module is used in similar situations differently. So I'd just unify imports for the `logger` module, particularly I'd use `import spef.utils.logger as logger` when there's a lot of names to import. The rest of the imports in this PR are fine.

Reply:
The imports should be fixed now.